extensions/scarb-doc/tests/data/json_output_test_data.json

```diff
@@ -141,8 +141,8 @@
                 "item_data": {
                   "name": "SHAPE_CONST",
                   "doc": "Constant for the shape type\n",
-                  "signature": null,
-                  "full_path": "Shape::SHAPE_CONST"
+                  "signature": "",
```

Comment:
This is on purpose - if sth changes in the compiler I will see it there. Otherwise I can just skip empty strings when generating docs